### DISCUSSION TITLE
Gazebo Plugin in docker container

### DISCUSSION THREAD

**Discussion by JannisWagner** [FieldRobotEvent/Virtual_Field_Robot_Event#48] [Q&A]
Hey all,

our own robot model uses Intel Realsense Depth Cameras. To simulate these we are using the realsense gazebo plugin by pal-robotics. This works fine if we run it on our local machines, but when we try to implement the robot into the docker container structure the depth-cameras won't start. The error in the a-container log is the following:
`[Err] [Plugin.hh:180] Failed to load plugin librealsense_gazebo_plugin.so: librealsense_gazebo_plugin.so: cannot open shared object file: No such file or directory
`
We inserted the plugin-package as well as our robot model into the my_robot directory.  The robot is getting spawned successfully, however the cameras are not working. Is there a way to make such a plugin work ?

Thank You & best regards 

Jannis

**Comment by ThijsRuigrok** [comment]
Hi, 

I think the easiest solution would be for me to install the plugin in the a_container. Can you provide the command to install this plugin? It would be perfect if you try this out in the Dockerfile for the a_container and post the edited Dockerfile here. I will then apply the changes to the a_container. 

Kind regards,
Thijs

**Reply by JannisWagner** [reply]
Hi Thijs,

thanks for your reply. When searching for a workaround we noted that the already installed kinect-plugin works for us as well. In addition we discovered that this plugin is less performance intensive and works with the docker environment out of the box. So we decided to change to this plugin and don't require the Realsense Plugin anymore.

Kind regards,
Jannis

**Reply by ThijsRuigrok** [reply]
Cool! Good job.
